src/Http/Controllers/Auth/SsoController.php

```diff
@@ -43,14 +43,36 @@ class SsoController extends Controller
      * Redirect the user to the Eve Online authentication page.
      *
      * @param \Laravel\Socialite\Contracts\Factory $social
+     * @param string $profile
      * @return mixed
      * @throws \Seat\Services\Exceptions\SettingException
      */
-    public function redirectToProvider(Socialite $social)
+    public function redirectToProvider($profile = null, Socialite $social)
     {
 
+        $scopes_setting = collect(setting('sso_scopes', true));
+
+        // Get the scopes that are marked as the default.
+        $scopes = $scopes_setting->first(function ($item) {
+            return $item->name == 'default';
```

Comment:
You add a `default` property to your object, why not using it instead the name ?
I think it would be more reliant ;)

And doing it so will allow you to determine which scope-profile is the default one.